cmake/Dependencies.cmake

```diff
@@ -456,12 +456,37 @@ endif()
 
 # ---[ OpenMP
 if(USE_OPENMP)
-  find_package(OpenMP)
+  set(WITH_OPENMP ON CACHE BOOL "OpenMP support if available?")
```

Comment:
When this block moved, did you change it at all, or is it unchanged?

Reply:
@ezyang 
The block I doesn't chage are from [line 960](https://github.com/pytorch/pytorch/blob/1637729620599706a8647f32fd375f623c4b7d9a/cmake/Dependencies.cmake#L960) to [line 985](https://github.com/pytorch/pytorch/blob/1637729620599706a8647f32fd375f623c4b7d9a/cmake/Dependencies.cmake#L985).